scripts/test/e2e/run

```diff
@@ -17,13 +17,33 @@ function setup {
     # TODO: only run if inside a container
     docker network connect "$network" "$(hostname)"
 
+    # [k8s] post setup dind
+    docker exec ${project}_engine_1 sh -c "
+            mkdir -p /var/lib/kubelet &&
+			mount -o bind /var/lib/kubelet /var/lib/kubelet &&
+			mount -o bind /run /run &&
+			mount --make-shared /var/lib/kubelet &&
+			mount --make-shared /run
+            mkdir -p /etc/cni /opt/cni/bin /var/log/pods /var/log/containers
+    "
+    # [k8s] fetch default cni plugins
+    # FIXME(vdemeester) put this into the Dockerfile.dev
+    # FIXME(vdemeester) pre-load images
+    docker cp /tmp/cni/bin ${project}_engine_1:/opt/cni
```

Comment:
I think it might be time to create a new `Dockerfile.test-e2e-runner` so we can add this, and don't have to add kubectl to `Dockerfile.dev`.  

It won't need vndr, bindata, or filewatcher, so there shouldn't be any overlap between them.